bundles/com.aptana.projects/src/com/aptana/projects/wizards/AbstractNewProjectWizard.java

```diff
@@ -744,6 +757,20 @@ protected IPath getDestinationPath()
 		return destPath;
 	}
 
+	private void deleteProject()
+	{
+		if(newProject.exists())
+		{
+			try
+			{
+				newProject.delete(true, null);
+			}
+			catch (CoreException e)
+			{
+			}
```

Comment:
Log the exception with some debug scope - so that we can track the issue when the troubleshooting is enabled. Otherwise will never know why the project delete is failed.